server/trafficrules/src/main/java/com/coffeejug/trafficrules/TrafficrulesApplication.java

```diff
@@ -0,0 +1,20 @@
+package com.coffeejug.trafficrules;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+
+@SpringBootApplication
+public class TrafficrulesApplication {
+
+    private static final Logger logger = LoggerFactory.getLogger(TrafficrulesApplication.class);
```

Comment:
Maybe `lombok` would be good fit here? :)

Reply:
I dislike lombok.

Reply:
But why? I allow reducing the boilerplate code

Reply:
We had an issue in Vaadin with toString, equals, hashCode from Lombok.
Getters and Setters are not a big problem.

Reply:
Well, I believe that Lombok is quite ok, but I'm not going to insist on using it